src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/DeferredContent/QuickInfoDisplayDeferredContent.cs

```diff
@@ -22,7 +23,8 @@ internal class QuickInfoDisplayDeferredContent : IDeferredQuickInfoContent
             IDeferredQuickInfoContent documentation,
             IDeferredQuickInfoContent typeParameterMap,
             IDeferredQuickInfoContent anonymousTypes,
-            IDeferredQuickInfoContent usageText)
+            IDeferredQuickInfoContent usageText,
+            IDeferredQuickInfoContent exceptionText)
```

Comment:
This will likely break TypeScript.  @vladima @paulvanbrenk , if this goes through, you'll have to provide an additional empty value for this as part of your quick info result.